src/components/QrScanner/ValidityCheckScreenContainer.tsx

```diff
@@ -11,27 +10,32 @@ import { replaceRouteFn } from "../../common/navigation";
 export const ValidityCheckScreenContainer: FunctionComponent<NavigationProps> = ({
   navigation
 }) => {
-  const document: SignedDocument = navigation.getParam("document");
+  const document: OAWrappedDocument = navigation.getParam("document");
   const isSavable: boolean = navigation.getParam("savable");
-  const verificationStatuses = useDocumentVerifier(document as SignedDocument);
+  const { statuses, verify } = useDocumentVerifier();
+
+  useEffect(() => {
+    verify(document);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
actually is this required? if we put `verify` and `document` inside will it result in a loop?

Reply:
Turns out this isn't necessary